dao/src/main/java/com/epam/esm/config/DaoTestConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.esm.config;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.springframework.context.annotation.*;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+
+@Profile("test")
+@Configuration
+@EnableTransactionManagement
+@PropertySource("classpath:hikari-test.properties")
+@ComponentScan(basePackages = {"com.epam.esm.dao", "com.epam.esm.mappers"})
+public class DaoTestConfig {
```

Comment:
I think this class should placed in test/java...

Reply:
commit e43bc4e